frontend/src/MyContext.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
file name is MyContext but it exports MyProvider. confusing

Reply:
I will try to explain:
The convention I saw in the tutorial do exactly that - Provider And Context in the same file.
I have split the classes to two separate files - MyProvider.js and MyContext.js. look how small is MyContext.js now:

```
import React from "react";

export const MyContext = React.createContext();
```
that's it. isn't it too small for a separate file?

Reply:
right, but the convention is that the file name is MyContext and not MyProvider? if so, let it be

Reply:
Yes and the reason is that you are using context from a provider or a consumer and for that you need this import:
`import { MyContext } from "./MyContext";`

for now i left it that way. I think using separate files will make it more clearer